CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ option(TXN_SIGNING_ENABLED "Enable External concord client transcattion signing"
 option(BUILD_COMM_TCP_PLAIN "Enable TCP communication" OFF)
 option(LEAKCHECK "Enable Address and Leak Sanitizers" OFF)
 option(THREADCHECK "Enable Thread Sanitizer" OFF)
+option(USE_CCACHE "Enable ccache to speed up recompilation" ON)
```

Comment:
this doesn't work with a native build in a conjunction with the `if (USE_CCACHE)` later in this file.
I propose not to change `CMakeLists.txt` at all, but rather call it as recommended `cmake -DCMAKE_CXX_COMPILER_LAUNCHER=ccache ..` from an outside `Makefile`